extensions/adobe/experience/customerJourneyManagement/message-interaction.schema.json

```diff
@@ -115,6 +115,112 @@
                 "internal": "Internal for AJO Landing Page",
                 "external": "External for Non-AJO Landing Page"
               }
+            },
+            "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/formData": {
+              "title": "Form Data",
+              "type": "object",
+              "description": "Data filled by the user on the landing page form.",
+              "properties": {
+                "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/subscriptions": {
+                  "title": "Subscription details.",
+                  "type": "object",
+                  "description": "Subscription details input by the end user in the landing page.",
+                  "meta:xdmType": "map",
+                  "additionalProperties": {
+                    "type": "object",
+                    "description": "Subscription details input by the end user in the landing page. ",
+                    "properties": {
+                      "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/val": {
+                        "title": "Subscription Choice",
+                        "type": "string",
+                        "description": "Subscription Choice for the Subscription List.",
+                        "enum": ["y", "n", "p", "u"],
+                        "meta:enum": {
+                          "y": "Subscribed to Subscription List",
+                          "n": "Unsubscribed to Subscription List",
+                          "p": "Subscription Pending",
+                          "u": "Subscription Unknown"
+                        }
+                      },
+                      "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/type": {
+                        "title": "Subscription Type",
+                        "type": "string",
+                        "description": "The type of subscription."
+                      }
+                    }
+                  }
+                },
+                "https://ns.adobe.com/experience/customerJourneyManagement/messageInteraction/consents": {
```

Comment:
Did you consider using the standard XDM "Consent and preferences Details" field group ? It will be great to use s a standard way to use consent signals consistently across the platform.